### PR TITLE
Ems 124235441 - Updated scope for planned activities in participant view.

### PR DESCRIPTION
**Pull request by eschlange**
- Added model scope.
- Updated content provider view.
- Added fixture to test new scope.
- Added spec for new scope.

[#124235441]

### CODE REVIEW

File: app/models/content_providers/unplanned_activity_form.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ def render_current(options)
                                   .view_context
                                   .current_participant
                                   .activities
-                                  .in_the_future
+                                  .starts_in_the_future
```
Comment (michaelwehrley) [inline app/models/content_providers/unplanned_activity_form.rb]: So we are no longer showing activities that end in the future, but have already started?

Comment (eschlange) [inline app/models/content_providers/unplanned_activity_form.rb]: That is correct, see Chris's response here: https://www.pivotaltracker.com/story/show/124235441

Comment (michaelwehrley) [inline app/models/content_providers/unplanned_activity_form.rb]: Awesome!  Thanks.

**Comment by michaelwehrley** [comment]
👍

**Pull request merged**